cmd/template/framework/files/routes/echo.go.tmpl

```
@@ -34,3 +42,33 @@ func (s *Server) healthHandler(c echo.Context) error {
 	return c.JSON(http.StatusOK, s.db.Health())
 }
 {{end}}
+
+{{if .AdvancedOptions.Websocket}}
+func (s *Server) websocketHandler(c echo.Context) error {
+	w := c.Response().Writer
+	r := c.Request()
+	socket, err := websocket.Accept(w, r, nil)
+
+	if err != nil {
+		log.Print("could not open websocket")
+		_, _ = w.Write([]byte("could not open websocket"))
+		w.WriteHeader(http.StatusInternalServerError)
+		return nil
```

Same here - we need to surface the error